### PR TITLE
fix: speed up local evaluation endpoint

### PR DESCRIPTION
**Pull request by neilkakkar**
## Problem

Removes unnecessary cruft from local evaluation

## Changes

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: posthog/api/test/test_feature_flag.py [hunk 1]
```diff
@@ -1033,10 +1033,7 @@ def test_local_evaluation(self, mock_capture):
                 },
                 "deleted": False,
                 "active": True,
-                "is_simple_flag": True,
-                "rollout_percentage": 20,
                 "ensure_experience_continuity": False,
-                "experiment_set": [],
```
Comment (liyiy) [inline posthog/api/test/test_feature_flag.py:1039]: The frontend is using `experiment_set` to determine if a feature flag is linked to an experiment, how much faster does removing this make the local eval?

Comment (neilkakkar) [reply posthog/api/test/test_feature_flag.py:1039]: This endpoint isn't called by the frontend, it's for client libraries, so nothing needs to change for the frontend :)

Comment (liyiy) [reply posthog/api/test/test_feature_flag.py:1039]: We can probably just make an extra call to experiments in the FE and filter for those feature flags if we have to

Comment (neilkakkar) [reply posthog/api/test/test_feature_flag.py:1039]: And 20-50% depending on the team, but the key is that this endpoint is called every 30 seconds by every client library out in the wild, so even small gains add up

Comment (neilkakkar) [reply posthog/api/test/test_feature_flag.py:1039]: These tests are simulating what's received by the client libraries

**Review by liyiy — APPROVED** [review]

File: posthog/api/test/test_feature_flag.py (hunk 1 above)
Comment (liyiy) [reply posthog/api/test/test_feature_flag.py:1039]: i see! very nice

**Pull request merged**